utils/android/adb/commands.py

```diff
@@ -146,7 +153,7 @@ def execute_on_device(executable_path, executable_arguments):
         # the executable on the device.
         return 1
 
-    print(stdout)
+    print(stdout, end='')
 
     shell(['rm', '-rf', uuid_dir])
```

Comment:
Not your fault, but eww.  Can we not use `os.removedirs` with `atexit.register`?

Reply:
This is not the local machine. This `shell` sends the `adb shell` to the Android device. The reason for not doing it on `atexit` is that in the case of failure you can investigate what happened in the files of the phone.